app/Http/Controllers/HomeController.php

```diff
@@ -13,12 +14,17 @@ class HomeController extends Controller
      * @return \Illuminate\Http\Response
      */
     public function index()
-    {
+    {   
+        $recent_projects = Expense::orderBy('created_at','DESC')
+                ->skip(0)
+                ->take(2)
```

Comment:
The task specification states that 4 projects should be displayed not 2

Reply:
Alright sir.  Done